cobalt/build/configs/linux-x64x11/args.gn

```diff
@@ -46,3 +46,10 @@ enable_nacl = false
 enable_print_preview = false
 use_cups = false
 use_kerberos = false
+
+include_transport_security_state_preload_list = false
+chrome_root_store_optional = false
+build_epoll_based_tools = false
+enable_unix_sockets = false
+use_remote_test_server = false
+# enable_built_in_dns = false?
```

Comment:
We should preferably keep it in, if possible. Unless the dependency and modification footprint is too large.